docs/capabilities.md

```diff
@@ -114,26 +114,26 @@
 * `chat-keep-notifications` - Whether messages can be retrieved without marking notifications as read
 
 ## 17
-* `avatar` - Avatar of conversation
+* `avatar` (local) - Avatar of conversation
```

Comment:
There was an open question in the issue if it should be remote, since we load the image from there? Current implementation on iOS uses the local capability, though.